### ISSUE TITLE
Plugin does not support root paths.

### ISSUE THREAD

**Issue by wsalembi** [kongchen/swagger-maven-plugin#217]
JAX-RS to Swagger generates a swagger.json file not valid according to the specification.

```
@Path("/")
@Api
public class MyResource {

    @GET
    @ApiOperation("List all possible links in my API")
    public Response links() {
        return Response.ok("...").build();
    }
}
```
generates the following Swagger fragment

```
  "paths" : {
    "" : {
      "get" : {
        "summary" : "List all possible links in my API",
        "description" : "",
        "operationId" : "links",
        "responses" : {
          "default" : {
            "description" : "successful operation"
          }
        }
      }
```

The path should start with a slash. See http://swagger.io/specification/#pathsObject

The error seems to be located in 
com.github.kongchen.swagger.docgen.reader.AbstractReader.parseOperationPath(String, Map<String, String>) which transforms "/" to ""

**Comment by who** [comment]
I've fixed this in 3.1.1-SNAPSHOT. Could you please verify ?

**Comment by who** [comment]
Hi @wsalembi ,
I've fixed this in 3.1.1-SNAPSHOT. Could you please verify ?

**Comment by who** [comment]
@wsalembi Please re-open if you still see this issue.

Use the latest version - swagger-maven-plugin 3.1.2

**Comment by wsalembi** [comment]
I confirm fixed in 3.1.2
